src/math/ceil.js

```diff
@@ -0,0 +1,35 @@
+const ceil = (number, precision) => {
+    let numberString = number.toString();
+    let decimalIndex = numberString.indexOf('.');
+    let beforeDecimal = numberString.substring(0, decimalIndex);
+    let afterDecimal = numberString.substring(decimalIndex, numberString.length);
+  if (Number.isInteger(number)){
+    return number;
+  } else if (Number.isInteger(number) === false) {
+      if (precision === undefined || precision  === 0) {
+        for(let i = 0; i < afterDecimal.length; i++){
+          if(numberString[i] > 0){
+            let roundUp = parseInt(beforeDecimal) + 1;
+            return roundUp;
+          }
+        }
+      } else if (precision > 0) {
```

Comment:
It makes your if ... else statement less verbose and the checks are less redundant.